### ISSUE TITLE
A11y violation of colour contrast in amp ads '.i-amphtml-new-loader-ad-label'

### ISSUE THREAD

**Issue by LilyL0u** [bbc/simorgh#9580]
![image (6)](https://user-images.githubusercontent.com/54575620/138489802-2d2b2304-208c-47a6-bae0-98586c938522.png)

We are getting intermittent failures in our E2Es on an a11y violation on amp pages. It is for colour contrast and is happening on '.i-amphtml-new-loader-ad-label'. This is part of the amp ads. We do not control these, and the intermittent nature could be due to different ads showing. 

Example of a page this has happened on https://www.bbc.com/russian/features-54391793.amp

Or could it be if an ad doesn't render, it isn't liking the placeholder colour?
When I look at where the element is causing the violation
it is in
<style amp-extension=“amp-loader”>
rather than in
<style amp-extension=“amp-ad”> 
which sounds like it holds the ad itself

Not sure how it works though.

Here is the placeholder which would be behind the ad and shows if no ad rendered

<img width="1014" alt="Screenshot 2021-10-22 at 17 58 13" src="https://user-images.githubusercontent.com/54575620/138495079-e2087417-07f7-4583-bb2c-13763a3a878f.png">

We could stop this by exclusion, as in the previously used code 
`const exclude = [
  // These elements can contain a11y violations as we have no control over what is rendered inside of them
  '.bbc-news-vj-embed-wrapper, [id^="include-"]', // VJ includes
  '[class*=dotcom], [id*=dotcom], amp-ad', // GNL ads
];`

**Comment by DarioR01** [comment]
excluding the css class had no change. We are still getting this error

<img width="1061" alt="Screenshot 2021-10-28 at 11 56 38" src="https://user-images.githubusercontent.com/90621252/139244250-3bc2c41a-1265-4d8a-8ab2-b21f510348da.png">
